config/development.json

```diff
@@ -28,6 +28,7 @@
 	"rebrand_cities_prefix": "rebrandcitiesdevelopmentsite",
 	"livechat_support_locales": [ "en", "es", "pt-br" ],
 	"features": {
+		"no-force-sympathy": true,
```

Comment:
I think this is checked in accidentally right?
I've rejected based on this only, the other comments aren't blockers at all :)

Reply:
This is listed in the PR description so seems intentional. But I'm guessing if it's needed in `development` it's probably needed in the other environments too?

Reply:
This is here intentionally — apologies, I should have been clearer on what's going on here! Related to Phil's comment in https://github.com/Automattic/wp-calypso/pull/34770#pullrequestreview-266887926, in `development` only, the persisted state is randomly cleared 25% of the time.

Here's where it's set randomly: https://github.com/Automattic/wp-calypso/blob/81162c9e9c8c2be8e78cac04e4c8c8735963434a/client/state/initial-state.js#L77

This is where it gets called: https://github.com/Automattic/wp-calypso/blob/81162c9e9c8c2be8e78cac04e4c8c8735963434a/client/state/initial-state.js#L232

So the change only needs to happen in the development environment to prevent the state from being cleared. Because this PR relies on persisted state existing in order to do the transfer from logged out to logged in state, I thought it could be confusing in the future if we continued to have the state randomly cleared on some page loads. For example, if a couple of months from now another developer is testing the login behaviour locally and wonders why sometimes the redirect doesn't work, but sometimes it does. By using the feature flag, people can still get the clearing behaviour back if they want it, it just wouldn't be the default.

What do you think?

Reply:
Would it be crazy (or even work) to check the abtest here?

```
	if ( config.isEnabled( 'no-force-sympathy' ) || 'last' === abtest('moveUserStepPosition') ) {
		return false;
	}
```

The AB isn't permanent anyway.

Reply:
Nice way to do it @ramonjd! I'll try it out...

Reply:
Works nicely, I've updated it and removed the line from `development.json`, so I'll resolve this part of this discussion.